tsl/src/bgw_policy/job_api.c

```diff
@@ -380,6 +399,7 @@ job_alter(PG_FUNCTION_ARGS)
 		values[8] = CStringGetTextDatum(schema_qualified_check_name);
 	else
 		nulls[8] = true;
+	// values[9] = BoolGetDatum(job->fd.fixed_schedule);
```

Comment:
Shouldn't be possible to change the "fixed_schedule" flag in alter_job API?

Reply:
I had discussed with @jfjoly that it was not necessary to allow toggling this, but it's also not hard to do if we decide that we want to do this after all.

Reply:
Not doing that means that users after upgrade the extension including this feature should re-create their jobs (delete and add) in order to have fixed scheduled jobs.

Reply:
In any case would be good to remove this commented code